### PR TITLE
feat: Remove preact and use React instead

### PR DESCRIPTION
**Pull request by enguerran**
Some cleaning work still need to be done, but:

- [x] remove `preact` and `preact-portal`
- [x] remove webpack aliases
- [x] update `react`, `react-dom` and `react-styleguidist`
- [x] create a `Portal` component

### CODE REVIEW

File: react/Modal/Readme.md [hunk 1]
```diff
@@ -7,7 +7,8 @@ initialState = { modalDisplayed: false};
   <button onClick={()=>setState({ modalDisplayed: !state.modalDisplayed })}>
     Toggle modal
   </button>
-  {state.modalDisplayed && <Modal title='Ada Lovelace' description={content.ada.short} dismissAction={() => setState({ modalDisplayed: false })} /> }
+  {state.modalDisplayed && <Modal into="#modalContainer" title='Ada Lovelace' description={content.ada.short} dismissAction={() => setState({ modalDisplayed: false })} /> }
+  <div id="modalContainer" />
```
Comment (GoOz) [inline react/Modal/Readme.md]: You can't append the modal directly into the `body` anymore? :'(

**Comment by enguerran** [comment]
I don't know how to update the `Modal` component that uses `UNSAFE_componentWillUpdate`

https://github.com/cozy/cozy-ui/blob/3524f24fdd1514a9e6e4135e22d8769099019eee/react/Modal/index.jsx#L42-L45

This breaks eslint rules and I had to `push --no-verify`.

File: package.json [hunk 2]
```diff
@@ -65,13 +65,12 @@
     "node-polyglot": "^2.2.2",
     "npm-run-all": "^4.0.1",
     "postcss-loader": "^2.0.6",
-    "preact": "^8.1.0",
-    "preact-compat": "^3.13.1",
+    "prettier": "^1.12.1",
```
Comment (kosssi) [inline package.json]: Pourquoi avoir ajouté `prettier` ? C'est normalement `eslint-config-cozy-app` qui s'occupe de faire l'import de prettier.

File: react/Modal/Readme.md (hunk 1 above)
Comment (kosssi) [reply react/Modal/Readme.md]: Yes you can with `into="body"` you can see [here](https://github.com/cozy/cozy-ui/pull/594/files#diff-113f5e7400f32829864d3c8c2c2fe97eR4).

Comment (GoOz) [reply react/Modal/Readme.md]: Oh ok, the example misled me, thanks.

File: package.json (hunk 2 above)
Comment (enguerran) [reply package.json]: That is a side-effect from the rebase. I will remove it and use cozy-scripts configuration files.

**Comment by CPatchane** [comment]
Any updates about this PR?

**Pull request merged**